ui/src/authors/components/OrcidPushSetting.jsx

```diff
@@ -2,9 +2,8 @@ import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { Switch, Popconfirm } from 'antd';
 import OrcidPushSettingMessageContainer from '../containers/OrcidPushSettingMessageContainer';
-import ExternalLink from '../../common/components/ExternalLink.tsx';
 
-function renderPopConfirmTitle(isCurrentlyEnabled, authorBAI) {
+function renderPopConfirmTitle(isCurrentlyEnabled) {
```

Comment:
this function needs the `authorBAI` see line 40

Reply:
That function has authorBAI on line 25

Reply:
![Screenshot 2021-07-20 at 09 23 56](https://user-images.githubusercontent.com/4287772/126279153-329ed7d1-1fdd-4fd1-b5a5-96fa7d6c344d.png)

You have to change this as well because you changed the signature of the function

Reply:
updated